shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -127,6 +127,9 @@ public void unlockPlatformViewInputConnection() {
      * The TextInputPlugin instance should not be used after calling this.
      */
     public void destroy() {
+        if (inputTarget.type != InputTarget.Type.NO_TARGET) {
```

Comment:
We should only do this when the target is a FRAMEWORK_CLIENT (if the target is a PLATFORM_VIEW `_currentConnection` on the framework side is probably null).